tools/dump_tool/HowToUseDumpTool.md

```diff
@@ -30,6 +30,11 @@ Debug Options -> Enable Debug Output Features to y
 ```
 Debug Options -> Enable backtracking using Frame pointer register  to y
 ```
+```
+NOTE: - For devices that use ARM Cortex M, backtracking of frame pointer is not supported.
+Hence, it is not possible to get the pointer to stack frame for these devices.
```

Comment:
I think following would be more appropriate:
"it is not possible to obtain exact call stack for the crash"

Reply:
This can be modified in the commit description as well

Reply:
Ohkay, thankyou.
Updated